sdk/identity/Azure.Identity/src/MsalClientBase.cs

```diff
@@ -35,13 +35,13 @@ protected MsalClientBase(CredentialPipeline pipeline, string tenantId, string cl
             _ensureInitAsync = new Lazy<Task>(InitializeAsync);
         }
 
-        protected string TenantId { get; }
+        internal string TenantId { get; }
```

Comment:
Are these internal for testing purposes?

Reply:
Yes, I made them internal so that they could be verified in tests.